docs/source/libraries/sklearn.rst

```diff
@@ -0,0 +1,225 @@
+* AdaBoostClassifier_ (only :func:`eli5.explain_weights`)
+* AdaBoostRegressor_ (only :func:`eli5.explain_weights`)
+* RandomForestClassifier_
+* RandomForestRegressor_
+* ExtraTreesClassifier_
+* ExtraTreesRegressor_
+
+For ensembles :func:`eli5.explain_weights` computes feature importances
+and their std deviation.
+
+:func:`eli5.explain_prediction` is less straightforward for ensembles and
+trees; eli5 uses an approach based on ideas from
+http://blog.datadive.net/interpreting-random-forests/ :
+feature weights are calculated by following decision paths in trees
+of an ensemble (or a single tree for DecisionTreeClassifier and
+DecisionTreeRegressor). Each node of the tree has an output score, and
+contribution of a feature on the decision path is how much the score changes
+from parent to child.
+
+.. note::
+    There is a separate package for this explaination method
+    (https://github.com/andosa/treeinterpreter); eli5 implementation
+    is independent.
+
+.. _DecisionTreeClassifier: http://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeClassifier.html#sklearn.tree.DecisionTreeClassifier
+.. _DecisionTreeRegressor: http://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeRegressor.html#sklearn.tree.DecisionTreeRegressor
+.. _GradientBoostingClassifier: http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html#sklearn.ensemble.GradientBoostingClassifier
+.. _GradientBoostingRegressor: http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html#sklearn.ensemble.GradientBoostingRegressor
+.. _AdaBoostClassifier: http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.AdaBoostClassifier.html#sklearn.ensemble.AdaBoostClassifier
+.. _AdaBoostRegressor: http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.AdaBoostRegressor.html#sklearn.ensemble.AdaBoostRegressor
+.. _RandomForestClassifier: http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html#sklearn.ensemble.RandomForestClassifier
+.. _RandomForestRegressor: http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html#sklearn.ensemble.RandomForestRegressor
+.. _ExtraTreesClassifier: http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesClassifier.html#sklearn.ensemble.ExtraTreesClassifier
+.. _ExtraTreesRegressor: http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesRegressor.html#sklearn.ensemble.ExtraTreesRegressor
+
+Reversing hashing trick
+-----------------------
+
+eli5 allows to recover feature names for HashingVectorizer_ and FeatureHasher_
+by computing hashes for the provided example data.
+:func:`eli5.explain_prediction` handles HashingVectorizer as ``vec``
+automatically; to handle HashingVectorizer_ and FeatureHasher_ for
+:func:`eli5.explain_weights` use
+:class:`~.InvertableHashingVectorizer` or
+:class:`~.FeatureUnhasher`.
```

Comment:
Yeah, a good point. I hoped that a link to InvertableHashingVectorizer is enough (there is usage information in a docstring), but an example right here would be better indeed. I'll add it.